### ISSUE TITLE
Anything about watch on files editing?

### ISSUE THREAD

**Issue by frederikhors** [magefile/mage#215]
Anything about watch on files editing?

**Comment by natefinch** [comment]
Not really.  Mage runs when you ask it to run. To "watch" files implies a long running process.  If you just care about last modified times, the `target` package can help.
